sprout/appliances/views.py

```diff
@@ -209,7 +208,7 @@ def versions_for_group(request):
                 for version in Template.get_versions(
                     container_q,
                     template_group=group, ready=True, usable=True, exists=True,
-                    preconfigured=preconfigured, provider__working=True, provider__disabled=False,
```

Comment:
Hmm, you cannot remove the preconfigured boolean from these queries, sometimes preconfigured appliances are not available and pure ones are, which would make confusion over failing requests.